Homework_7_Java_Arrays_Loops/JavaArraysAndLoops/src/StrongestRobot.java

```diff
@@ -0,0 +1,15 @@
+public class StrongestRobot {
+
+    public static void main(String[] args) {
+
+        int[] weights = {300, 450, 400, 500, 350};
+        int max = weights[0];
+        for (int i = 1; i < weights.length; i++) max = Math.max(max, weights[i]);

```

Comment:
It works, but could be solved without using Math.max using the things that we already learned (we didn't learn Math.max yet)

Let's say it's generally accepted, but there's a possible improvement - don't use any Math functions from the standard library, only operators, conditionals, arrays and loops